skyportal/tests/models/test_permissions.py

```diff
@@ -1,383 +1,446 @@
 def test_user_create_public_group(  # noqa
-    user, public_group,
+    user,
+    public_group,
 ):
     accessible = public_group.is_accessible_by(user, mode="create")
     assert accessible
 
 
 def test_user_create_public_groupuser(
-    user, public_groupuser,
+    user,
```

Comment:
is this black's formatting? I would expect such a short parameter list to all be on the above line

Reply:
Yeah, black makes no changes when I run 

```
(skyportal2) Dannys-MBP:skyportal danny$ black skyportal/tests/models/test_permissions.py
All done! ✨ 🍰 ✨
1 file left unchanged.
```